administrator/components/com_media/Controller/Api.php

```diff
@@ -151,7 +165,7 @@ public function files()
 			// Return the data
 			$this->sendResponse($data);
 		}
-		catch (MediaFileAdapterFilenotfoundexception $e)
+		catch (FileNotFoundException $e)
```

Comment:
Can you make a pr with this fix against the milestone1 branch? We need to have it fixed there as well.